core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-ema-bookmarks/dot-ema-bookmarks.component.html

```diff
@@ -1,7 +1,15 @@
-<p-button
-    (onClick)="toggleBookmark()"
-    [loading]="loading()"
-    [icon]="bookmarked() ? 'pi pi-star-fill' : 'pi pi-star'"
-    [label]="'editpage.toolbar.bookmark' | dm"
-    styleClass="p-button-text p-button-sm"
-    data-testId="bookmark-button" />
+@if (variant === TOOLBAR_VARIANTS.NEW_UVE) {
+    <p-button
+        [icon]="bookmarked() ? 'pi pi-star-fill' : 'pi pi-star'"
+        [loading]="loading()"
+        (click)="toggleBookmark()"
+        styleClass="p-button-text" />
+} @else {
+    <p-button
+        (onClick)="toggleBookmark()"
+        [loading]="loading()"
+        [icon]="bookmarked() ? 'pi pi-star-fill' : 'pi pi-star'"
+        [label]="'editpage.toolbar.bookmark' | dm"
+        styleClass="p-button-text p-button-sm"
+        data-testId="bookmark-button" />
+}
```

Comment:
Looks like we only need to hide the label and a class. Do we need 2 buttons small changes?

Reply:
Not really, but I think it's easier to read and grasp the difference between versions having two buttons